sessions/biases-in-delay-distributions.qmd

```diff
@@ -108,7 +107,15 @@ This makes handling the data easier - we don't have to make any conversions befo
 Each of the numbers now represent the number of days that have passed since the start of the outbreak.
 That is, each of the numbers correspond to a day.
 In that sense, the data is more like typical data we get from infectious disease outbreaks, where we would usually have a line list with key events such as symptom onset or death reported by a *date*.
-In statistical terms, we call the delay *double interval censored*: "double" because the delays represent the time between two events that are both censored; and "interval" because all we know about the timings of the events is that they happened in a certain time interval (between 0:00 and 23:59 on the recorded day).
+In statistical terms, we call the data *double interval censored*:
```

Comment:
(if you haven't yet) double check this doesn't need a new line for it to be recognised as bulleted list (I thought it did but was wrong on the updated quarto slides)